src/TouristInfoCards.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+function TouristInfoCards() {
+  const touristCardData = [
+    {
+      image:
+        "https://images.pexels.com/photos/12319481/pexels-photo-12319481.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1",
+      city: "Glasgow",
+      description:
+        "Ancestor laugh fat lit chair running Uruks master loyal ready. Explanation Dori aside. Frailty within Witch-king greedily. What about second breakfast? ",
```

Comment:
Ahhhh Lord Of The Rings Lorem Ipsum!! I hadn't heard of this before :)

Reply:
It's awsome!!